scripts/etc/spark-defaults.conf.example

```diff
@@ -0,0 +1 @@
+spark.master local[*]
```

Comment:
Can you create a separate path for this apart from scripts/etc/* , since thats used for presto and I would like to keep this distinct . Maybe scripts/spark/etc/.. and scripts/spark/... in general for spark related scripts (apart from docker file) ?

Reply:
Thanks for your suggestion. I created separate directories to store the scripts used by presto and spark.